demos/python_demos/action_recognition/README.md

```diff
@@ -69,9 +68,10 @@ To run the demo, you can use public or pre-trained models. To download the pre-t
 
 **For example**, to run the demo for in-cabin driver monitoring scenario, please provide a path to the encoder and decoder models, an input video and a file with label names:
 ```bash
-python3 demo.py --encoder models/driver_action_recognition_tsd_0002_encoder.xml \
-    --decoder models/driver_action_recognition_tsd_0002_decoder.xml \
-    --labels driver_actions.txt
+python3 demo.py -m_en models/driver_action_recognition_tsd_0002_encoder.xml \
```

Comment:
Here and above `demo.py` is still in use.

Reply:
Changed